index.html

```diff
@@ -260,11 +260,12 @@ <h1>Contact me</h1>
                 I'd love to help with it </p>
             </div>
             <div class="contact-form">
-            <form action="https://formspree.io/html/" method="post">
+            <form action="https://formspree.io/f/xnqwordb" method="post">
                 <input name="first_name" id="text" type="text" maxlength="30" placeholder="Afolabi" required>
                 <input name="Email" id="email" type="email" placeholder="example@gmail.com" required>
-                <input type="text" id="message" maxlength="500" placeholder="write your message here" required>
-                <button type="submit" class="btn">Get It Touch</button>
+                <textarea id="message" name="message" maxlength="500" required>write your message here
+                </textarea>
+                <button type="submit" class="btn">Get in Touch</button>
```

Comment:
- [ ] Please add some left padding to your input fields and make sure it matches with the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/166664913-2a2ed05a-cef0-488c-bb51-abe9bfcc9b79.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166664937-98cbb4cd-3a76-4a61-a551-d98f21303ffb.png)